components/button/button-add.js

```diff
@@ -54,13 +58,18 @@ class ButtonAdd extends RtlMixin(PropertyRequiredMixin(FocusMixin(LocalizeCoreEl
 				justify-content: center;
 				outline: none;
 				padding: 0;
+				position: relative;
 				user-select: none;
 				white-space: nowrap;
 				width: 100%;
+				z-index: 10;
```

Comment:
The only way I found to get the `button-add` to be able to be interacted with from below (i.e., from a sibling list item) and also for the tooltip to be above neighbouring `button-add`s was to do this. I think this is relatively safe, as `button-add` would generally usually be above nearby content, but it could cause problems

Reply:
This is the sticky header case with `z-index` > 5:
![image](https://github.com/BrightspaceUI/core/assets/9042472/b9bb7894-78f8-4012-8d13-b0eb0a82b774)

Without the `z-index`, it's possible for tooltip to float over the sticky header. That might have been an existing issue we were just "living with", as it probably would not come up often.
![image](https://github.com/BrightspaceUI/core/assets/9042472/54367a67-5aed-4d40-85d4-1bced6351660)

Is there a reason `10` was chosen? It seems like `1` might be sufficient to stack it on top of the adjacent next sibling, while allowing it to float under the sticky header. But it looks like drag & drop uses `z-index` a bit, so maybe `1` _isn't_ sufficient to avoid issues.

Reply:
This is kinda icky...

![image](https://github.com/BrightspaceUI/core/assets/9042472/826edbcd-8578-4c3d-8253-0dd699337c62)

But like the 2nd screenshot above, it's not due to your `z-index` usage, but rather, I think it's probably just due to `d2l-tooltip`. I think it was possible to make this happen without your changes, it just would have been less likely because it would require users to put tooltips (or components using them) into their items, and many list-items do not contain interactive content.

Reply:
Ah gross - I'll play around with smaller z-indexes and see if I can get something else working. Agreed with your analysis about tooltip - it has a large z-index on it that's causing this to happen and this just makes it a bit more likely (though hopefully this is still rare in practice)

Reply:
z-index of 1 does seem to give the functionality we want and does not have that sticky header issue. I didn't see any drag and drop issue (we do hide the add button during dragging so it hopefully wouldn't have an effect) but let me know if there are any you know of.